packages/react-inbox/src/components/Messages2.0/index.tsx

```diff
@@ -153,6 +153,7 @@ const Messages: React.ForwardRefExoticComponent<
     },
     ref
   ) => {
+    const [focusedMessageId, setFocusedMessageId] = useState<string>();
```

Comment:
useState("") generally safer and won't need the type annotation

Reply:
ill do both